timescaledb/quick-start/java.md

```diff
@@ -0,0 +1,621 @@
+                    )
+                    """);
+        }
+
+        try (var stmt = conn.createStatement()) {
+            stmt.execute("""
+                    CREATE TABLE sensor_data (
+                        time TIMESTAMPTZ NOT NULL,
+                        sensor_id INTEGER REFERENCES sensors (id),
+                        temperature DOUBLE PRECISION,
+                        cpu DOUBLE PRECISION
+                    )
+                    """);
+        }
+
+        try (var stmt = conn.createStatement()) {
+            stmt.execute("SELECT create_hypertable('sensor_data', 'time')");
+        }
+    }
+
+    private static void insertData(final Connection conn) throws SQLException {
+        final List<Sensor> sensors = List.of(
+                new Sensor("a", "floor"),
+                new Sensor("a", "ceiling"),
+                new Sensor("b", "floor"),
+                new Sensor("b", "ceiling"));
+        for (final var sensor : sensors) {
+            try (var stmt = conn.prepareStatement("INSERT INTO sensors (type, location) VALUES (?, ?)")) {
+                stmt.setString(1, sensor.type());
+                stmt.setString(2, sensor.location());
+                stmt.executeUpdate();
+            }
+        }
+
+        final var sensorDataCount = 100;
+        final var insertBatchSize = 10;
+        try (var stmt = conn.prepareStatement("""
+                INSERT INTO sensor_data (time, sensor_id, temperature, cpu)
+                VALUES (
+                    generate_series(now() - INTERVAL '24 hours', now(), INTERVAL '5 minutes'),
+                    floor(random() * (3) + 1)::INTEGER,
+                    random() * 100,
+                    random()
+                )
+                """)) {
+            for (int i = 0; i < sensorDataCount; i++) {
+                stmt.addBatch();
+
+                if ((i > 0 && i % insertBatchSize == 0) || i == sensorDataCount - 1) {
+                    stmt.executeBatch();
+                }
+            }
+        }
+    }
+
+    private static void executeQueries(final Connection conn) throws SQLException {
+        try (var stmt = conn.prepareStatement("""
+                SELECT time_bucket('15 minutes', time) AS bucket, avg(cpu)
+                FROM sensor_data
+                JOIN sensors ON sensors.id = sensor_data.sensor_id
+                WHERE sensors.type = ? AND sensors.location = ?
+                GROUP BY bucket
+                ORDER BY bucket DESC
+                """)) {
+            stmt.setString(1, "a");
+            stmt.setString(2, "floor");
+
+            try (var rs = stmt.executeQuery()) {
+                while (rs.next()) {
+                    System.out.printf("%s: %f%n", rs.getTimestamp(1), rs.getDouble(2));
+                }
+            }
+        }
+    }
+
+    private record Sensor(String type, String location) {
+    }
+}
+```
+
+Congratulations, you've successfully executed a query on TimescaleDB using Java and PostgreSQL JDBC.
```

Comment:
Missing the 🎉 Emoji 😄

Reply:
Before that, there was an exclamation mark at the end of the sentence, but @Loquacity removed it. I thought that excessive impression is not welcome. Although, of course, I would add emoji 😆

Reply:
I'm a tech writer, I thought the "Congratulations" was probably exuberant enough 😂  I don't mind what we choose, though, as long as it's consistent everywhere 👍